src/data_provider/src/osinfo/sysOsInfoWin.cpp

```diff
@@ -104,6 +104,25 @@ static std::string getBuild()
     return build;
 }
 
+static std::string getBuildRevision()
+{
+    std::string buildRevision;
+
+    if (IsWindows8OrGreater())
```

Comment:
Just a suggestion, and I realize this is nitpicking, but I think functions like these can be shorter and simpler if you use early returns, like this:
```
static std::string getBuildRevision()
{
    if (!IsWindows8OrGreater())
        return "";
    Utils::Registry currentVersion{HKEY_LOCAL_MACHINE, R"(SOFTWARE\Microsoft\Windows NT\CurrentVersion)", KEY_READ|KEY_WOW64_64KEY};
    auto ubr = currentVersion.dword("UBR", ubr);
    return ubr ? std::to_string(ubr) : "";
}

```